source/UF/Replacement.lagda

```diff
@@ -0,0 +1,108 @@
+Ian Ray, 17th August 2025.
+
+The type theoretic axiom of replacement (section 24.5 of Introduction to
+Homotopy Type Theory by Egbert Rijke:
+https://ulrikbuchholtz.dk/hott1920/hott-intro.pdf)
+is a statement about the size of the image of a function when certain smallness
+assumptions are imposed on the domain and codomain. The nomenclature is derived
+from the set theoretic axiom of replacement, where 'sethood' provides a notion
+of size. In type theory, this statement may be assumed or proven depending on
+the context. Thus, we will call it 'type replacement' or simply 'replacement'
+when there is no risk of confusion with set replacement (see Size.lagda).
+
+The statement of type replacement is as follows:
+The image of a map f : A → X, from a small type A to a locally small type X, is
+itself small.
+
+Note: some authors use the term 'essentially small' for what the TypeTopology
+library refers to as simply 'small'. Although, often we wish to consider size
+relative to an explicit universe, as you can see below.
+
+Type replacement is provable in the presence of a certain class of higher
+inductive types (HITs). In particular, "The Join Construction" by Egbert Rijke
+(https://arxiv.org/abs/1701.07538.) uses 'graph quotients' to give an
+alternative construction of the image where type replacement can be proven.
+More conservatively one may carry out this construction merely with pushouts
+(in fact, one only requires the join of maps and sequential colimits, which are
+instances of pushouts). This route is actively being explored in other
+TypeTopology files.
+
+It is worth noting that the status of type replacement's strength relative to
+other HITs is not completely understood, but it appears to be weaker than the
```

Comment:
Remove "other" as type replacement itself is not a HIT. Maybe also rephrase to "relative to the strength of adding certain HITs".